API/Program.cs

```diff
@@ -187,13 +187,12 @@ await context.HttpContext.Response.WriteAsync(
     // Configure the rate limit partitioning rules
     options.GlobalLimiter = PartitionedRateLimiter.Create<HttpContext, string>(context =>
     {
-        // Shared partition for anonymous requests
-        if (context.User.Identity is { IsAuthenticated: false })
+        // Unlimited partition for anonymous requests, as doing so
+        // would cripple the ability for the platform to support
+        // an influx of legitimate anonymous requests
+        if (context.User.Identity is null or { IsAuthenticated: false })
```

Comment:
```suggestion
        if (context.User.Identity is { IsAuthenticated: false })
```

null check here is not necessary, the pattern check will fail if it is null